styles/styles.css

```diff
@@ -0,0 +1,78 @@
+body {
+    background-color: #e2ebee;
+}
+
+
+header {
+    display: flex;  
+    justify-content: space-between;
+    align-items: center;
+    padding: 5px;
+    background-color: rgb(192, 207, 216);  
+    height: 15vh;
+}
+
+#logo {
+    display: flex;
+    order: -1;
+    cursor: pointer;
+    height: 22vh;
+}
+
+
+/* How were my bullet points already hidden? */
```

Comment:
An effect of `display: inline-block;` is hiding the bullet points! Previously the li default display value is `display: list-item;` and your styling overwrote it 👍🏻